### DISCUSSION TITLE
Unable to apply for more than 1 Octernship.

### DISCUSSION THREAD

**Discussion by CoffeeGeek101** [community/community#50027] [GitHub Education] [answered]
I have applied to a single octernship, and want to apply to another, the Octernship dashboard says, I can still apply for 3 more project, why can't I apply?
I really want to apply, because the deadline is nearing, can anyone help me ASAP.

**Comment by Anmol-Baranwal** [comment, marked as answer]
You can apply for another project once you have finished your first application. This is ideal for a large number of applicants. Please apply carefully, as you can only apply to a maximum of four projects per year.

**Reply by Anmol-Baranwal** [reply]
@CoffeeGeek101 
I just wanted to check in and see if you had any further questions or concerns about my answer on Github. If you have no further doubts or if your doubts have been resolved, would you mind marking my answer as accepted?

It would be greatly appreciated and it will also help other users who might be looking for a solution to a similar issue.
